checks/evaluation/permissions/permissions.go

```diff
@@ -99,77 +143,215 @@ func applyScorePolicy(results *checker.TokenPermissionsData, c *checker.CheckReq
 				loc.Snippet = newStr(r.File.Snippet)
 			}
 		}
-
 		text, err := createText(r)
 		if err != nil {
-			return checker.MinResultScore, err
+			return findings, err
```

Comment:
should this be `return nil, err`? Typically we return the zero value on error

Reply:
https://github.com/ossf/scorecard/pull/3693/commits/ef8518e2e5340b83c617cda46cd649e004d5b10f